package.json

```diff
@@ -4,120 +4,39 @@
   "description": "Concordium Network Dashboard",
   "main": "dashboard-backend.js",
   "engines": {
-    "node": ">=12"
+    "node": ">=14",
+    "npm": ">=7.20"
   },
   "scripts": {
-    "test": "jest",
     "clean": "rm -rf dist",
-    "lint": "tslint -c tslint.json 'src/**/*.ts' 'src/**/*.tsx'",
-    "start": "node dashboard-backend.js",
-    "start:prod": "NODE_ENV=production node dashboard-backend.js",
-    "build-client": "NODE_ENV=production webpack -p && cp -rp assets dist/public",
-    "dev-client": "webpack-dev-server -w",
-    "dev-client:hot": "webpack-dev-server -w --hot",
-    "build-server": "tsc -p ./src/server",
-    "dev-server": "tsc-watch -p ./src/server --onSuccess \"node dashboard-backend.js\"",
-    "dev": "npm run clean && concurrently --prefix \"[{name}]\" --names \"server,client\" -c \"blue,magenta\" \"npm run dev-server\" \"npm run dev-client\"",
-    "dev:hot": "npm run clean && concurrently --prefix \"[{name}]\" --names \"server,client\" -c \"blue,magenta\" \"npm run dev-server\" \"npm run dev-client:hot\"",
-    "build": "npm run clean && concurrently --prefix \"[{name}]\" --names \"server,client\" -c \"blue,magenta\" \"npm run build-server\" \"npm run build-client\""
+    "formatcheck": "elm-format src/elm/ --validate",
+    "build": "npm run clean && NODE_ENV=production webpack && cp -rp public/* dist",
```

Comment:
It looks like webpack should be able to do copy like this as part of the build process: https://stackoverflow.com/questions/27639005/how-to-copy-static-files-to-build-directory-with-webpack